web/src/settings_account.ts

```diff
@@ -296,6 +296,19 @@ export function render_user_timezone_dropdown_widget(): void {
             event.preventDefault();
             event.stopPropagation();
             const selected = widget.value();
+            if (!selected) {
+                return;
+            }
+
+            const prev = user_settings.timezone;
+
+            try {
+                if (typeof widget.render === "function") {
+                    widget.render(selected);
```

Comment:
This can just be replaced be `widget.render()`. I don't get why this try-catch blocks are needed.